src/TailwindBuilder.php

```diff
@@ -115,6 +116,6 @@ public function getOutputCssContent(): string
 
     private function createBinary(): TailwindBinary
     {
-        return new TailwindBinary($this->tailwindVarDir, $this->projectRootDir, $this->binaryPath, $this->output);
+        return new TailwindBinary($this->tailwindVarDir, $this->projectRootDir, $this->binaryPath, $this->binaryVersion, $this->output);
```

Comment:
Hm, this will introduce a possible BC break, though not sure if we can neglect it and mention in the release, it's convenient to have those 2 args close to each other

Reply:
TailwindBinary is basically internal,, so mentioning it in the release is OK i guess.